src/utils/pskc.cpp

```diff
@@ -53,8 +59,13 @@ void Pskc::SetSalt(const uint8_t *aExtPanId, const char *aNetworkName)
     cur += OT_EXTENDED_PAN_ID_LENGTH;
 
     VerifyOrExit(strlen(aNetworkName) > 0, ret = kPskcStatus_InvalidArgument);
-    memcpy(mSalt + cur, aNetworkName, strlen(aNetworkName));
-    cur += strlen(aNetworkName);
+
+    remainingSpace = sizeof(mSalt) - cur;
+    if (remainingSpace > 0) {
```

Comment:
To resolve pretty-check: https://github.com/openthread/ot-br-posix/actions/runs/6590257499/job/17915946140?pr=2072
```suggestion
    if (remainingSpace > 0)
    {
```